auth/models.py

```diff
@@ -10,7 +10,7 @@ class UserProfile(models.Model):
     # extra metadata
     dob = models.DateField(verbose_name='Date of Birth')
     car = models.BooleanField(verbose_name='Car?', default=False)
-    num_seats = models.PositiveSmallIntegerField(verbose_name='Number of Car seats', null=True)
+    num_seats = models.PositiveSmallIntegerField(verbose_name='Number of Car seats', null=True, blank=True)
 
```

Comment:
if they don't enter a value isn't it automatically null? take out the opportunity for them to have a blank option, but this should work too, you will have to explain it to me though, it seems a bit redundant.

Reply:
No, but good question. No, but good question. Check [this out](https://docs.djangoproject.com/en/dev/ref/models/fields/#field-options) for more info.